config/settings/base.py

```diff
@@ -132,3 +132,7 @@
 MEDIA_URL = '/mediafiles/'
 MEDIA_ROOT = os.path.join(BASE_DIR, 'mediafiles')
 
+# OSF Test user tokens. Used for unit tests
```

Comment:
Perhaps add...

Don't ever use these tokens to store non-public data as they are not secure.